Packages/Ink/Editor/Core/InkImporter.cs

```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+using UnityEditor.AssetImporters;
+using System.IO;
+using Ink.UnityIntegration;
+using UnityEditor;
+using Ink;
+
+/// <summary>
+/// Automatically compiles .ink assets each time they are imported, and creates an InkFile asset.
+/// </summary>
+[ScriptedImporter(1, "ink")]
+public class InkImporter : ScriptedImporter
+{
+    public override void OnImportAsset(AssetImportContext ctx)
```

Comment:
If compiling output of one ink file depends on another ink file then the [`AssetImportContext.DependsOnSourceAsset`](https://docs.unity3d.com/ScriptReference/AssetImporters.AssetImportContext.DependsOnSourceAsset.html) should be used for proper importer behavior.